diagnostic.md

```diff
@@ -39,31 +36,22 @@ methods.
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+The DOM is the place where all the action takes place. It acts as the middleman
+or translator between javascript and the different web browsers
```

Comment:
Not quite. The DOM is a mapping of the webpage, represented in the browser by the `document` object.